### ISSUE TITLE
Returns double the actual duration for ogg_vorbis

### ISSUE THREAD

**Issue by NinoSkopac** [JamesHeinrich/getID3#160]
Repro media file: https://drive.google.com/open?id=12Yy70HtCbOamHNdZpU47ZT-8prr6-I0w

code:
```php
$getId3 = new \getID3();
$this->file = $file; // @var \SplFileObject
$this->data = $getId3->analyze($this->file->getRealPath());
var_export($this->data);
```

output:
```
array (
  'GETID3_VERSION' => '1.9.13-201612051806',
  'filesize' => 676023,
  'filepath' => '/Users/ninoskopac/PhpstormProjects/Read2Me.online',
  'filename' => 'Google www-nytimes-com-2018-05-06-world-middleeast-israel-iran-nuclear-deal-html-d82d67 0.ogg',
  'filenamepath' => '/Users/ninoskopac/PhpstormProjects/Read2Me.online/Google www-nytimes-com-2018-05-06-world-middleeast-israel-iran-nuclear-deal-html-d82d67 0.ogg',
  'avdataoffset' => 118,
  'avdataend' => 674936,
  'fileformat' => 'ogg',
  'audio' => 
  array (
    'dataformat' => 'opus',
    'bitrate_mode' => 'vbr',
    'lossless' => false,
    'channels' => 1,
    'sample_rate' => 24000,
    'bitrate' => 15594.694351825974,
    'encoder' => 'Google Speech using libopus',
    'channelmode' => 'mono',
    'compression_ratio' => 0.04061118320788014,
    'streams' => 
    array (
      0 => 
      array (
        'dataformat' => 'opus',
        'bitrate_mode' => 'vbr',
        'lossless' => false,
        'channels' => 1,
        'sample_rate' => 24000,
        'bitrate' => 15594.694351825974,
        'encoder' => 'Google Speech using libopus',
        'channelmode' => 'mono',
        'compression_ratio' => 0.04061118320788014,
      ),
    ),
  ),
  'encoding' => 'UTF-8',
  'mime_type' => 'audio/ogg; codecs=opus',
  'ogg' => 
  array (
    'pageheader' => 
    array (
      0 => 
      array (
        'page_start_offset' => 0,
        'stream_structver' => 0,
        'flags_raw' => 2,
        'flags' => 
        array (
          'fresh' => false,
          'bos' => true,
          'eos' => false,
        ),
        'pcm_abs_position' => 0,
        'stream_serialno' => 0,
        'page_seqno' => 0,
        'page_checksum' => 2713804355,
        'page_segments' => 1,
        'page_length' => 19,
        'segment_table' => 
        array (
          0 => 19,
        ),
        'header_end_offset' => 28,
        'page_end_offset' => 47,
      ),
      'opus' => 
      array (
        'opus_magic' => 'OpusHead',
        'version' => 1,
        'out_channel_count' => 1,
        'pre_skip' => 312,
        'sample_rate' => 24000,
      ),
      1 => 
      array (
        'page_start_offset' => 47,
        'stream_structver' => 0,
        'flags_raw' => 0,
        'flags' => 
        array (
          'fresh' => false,
          'bos' => false,
          'eos' => false,
        ),
        'pcm_abs_position' => 0,
        'stream_serialno' => 0,
        'page_seqno' => 1,
        'page_checksum' => 4063806026,
        'page_segments' => 1,
        'page_length' => 43,
        'segment_table' => 
        array (
          0 => 43,
        ),
        'header_end_offset' => 75,
        'page_end_offset' => 118,
        'stream_type' => 'OpusTags',
      ),
      'eos' => 
      array (
        'page_start_offset' => 674936,
        'stream_structver' => 0,
        'flags_raw' => 4,
        'flags' => 
        array (
          'fresh' => false,
          'bos' => false,
          'eos' => true,
        ),
        'pcm_abs_position' => 8308278,
        'stream_serialno' => 0,
        'page_seqno' => 162,
        'page_checksum' => 4176698275,
        'page_segments' => 31,
        'page_length' => 1029,
        'segment_table' => 
        array (
          0 => 101,
          1 => 64,
          2 => 78,
          3 => 81,
          4 => 81,
          5 => 81,
          6 => 81,
          7 => 81,
          8 => 81,
          9 => 81,
          10 => 81,
          11 => 81,
          12 => 3,
          13 => 3,
          14 => 3,
          15 => 3,
          16 => 3,
          17 => 3,
          18 => 3,
          19 => 3,
          20 => 3,
          21 => 3,
          22 => 3,
          23 => 3,
          24 => 3,
          25 => 3,
          26 => 3,
          27 => 3,
          28 => 3,
          29 => 3,
          30 => 3,
        ),
        'header_end_offset' => 674994,
        'page_end_offset' => 676023,
      ),
    ),
    'vendor' => 'Google Speech using libopus',
    'comments_raw' => NULL,
    'samples' => 8308278,
    'bitrate_average' => 15594.694351825974,
    'encoding' => 'UTF-8',
  ),
  'opus' => 
  array (
    'opus_version' => 1,
    'sample_rate' => 24000,
    'out_channel_count' => 1,
  ),
  'playtime_seconds' => 346.17825,
  'bitrate' => 15594.694351825974,
  'playtime_string' => '5:46',
)
```

TLDR; getId3 returns playtime to be 5:46, however the actual duration is **exactly** half of that. I've reproduced on issue on ogg_vorbis files I've created using Ffmpeg (by concatenating multiple smaller files of the same type).

**Comment by NinoSkopac** [comment]
I'm using `1.9.13`, lemme try upgrading

**Comment by NinoSkopac** [comment]
yep, still broken

**Comment by NinoSkopac** [comment]
FFMPEG returns correct info:

```
$ ffmpeg -i Google\ www-nytimes-com-2018-05-06-world-middleeast-israel-iran-nuclear-deal-html-d82d67\ 0.ogg 

ffmpeg version 3.4.1 Copyright (c) 2000-2017 the FFmpeg developers
  built with Apple LLVM version 9.0.0 (clang-900.0.39.2)
  configuration: --prefix=/usr/local/Cellar/ffmpeg/3.4.1 --enable-shared --enable-pthreads --enable-version3 --enable-hardcoded-tables --enable-avresample --cc=clang --host-cflags= --host-ldflags= --enable-gpl --enable-libmp3lame --enable-libx264 --enable-libxvid --enable-opencl --enable-videotoolbox --disable-lzma
  libavutil      55. 78.100 / 55. 78.100
  libavcodec     57.107.100 / 57.107.100
  libavformat    57. 83.100 / 57. 83.100
  libavdevice    57. 10.100 / 57. 10.100
  libavfilter     6.107.100 /  6.107.100
  libavresample   3.  7.  0 /  3.  7.  0
  libswscale      4.  8.100 /  4.  8.100
  libswresample   2.  9.100 /  2.  9.100
  libpostproc    54.  7.100 / 54.  7.100
Input #0, ogg, from 'Google www-nytimes-com-2018-05-06-world-middleeast-israel-iran-nuclear-deal-html-d82d67 0.ogg':
  Duration: 00:02:53.09, start: 0.000000, bitrate: 31 kb/s
    Stream #0:0: Audio: opus, 48000 Hz, mono, fltp
At least one output file must be specified
```

**Comment by NinoSkopac** [comment]
Thanks for the fix man.
